pkg/shipper/shipper_e2e_test.go

```diff
@@ -224,7 +224,7 @@ func TestShipper_SyncBlocksWithMigrating_e2e(t *testing.T) {
 		defer upcancel2()
 		testutil.Ok(t, p.WaitPrometheusUp(upctx2, logger))
 
-		shipper := New(log.NewLogfmtLogger(os.Stderr), nil, dir, bkt, func() labels.Labels { return extLset }, metadata.TestSource, true, false, metadata.NoneFunc)
+		shipper := New(log.NewLogfmtLogger(os.Stderr), nil, dir, bkt, func() labels.Labels { return extLset }, metadata.TestSource, true, false, false, metadata.NoneFunc)
```

Comment:
I expect an E2E test to verify that there is no overlap check anymore.

Reply:
Yes, this will take me some time to work through.
Thanks!